source/documentation/doc-Release_Notes/topics/ref-Removed_Features_RHV.adoc

```diff
@@ -61,8 +61,6 @@ Removing this neither affects the ability to manage {virt-product-fullname} virt
 
 | Cockpit installation for Self-hosted engine| Using Cockpit to install the self-hosted engine is no longer supported. Use the command line installation.
 
-// Uncomment this line for RHV release that corresponds to oVirt 4.5.
-// Also, remove Deprecation notice for this feature for same release.
-// | oVirt Scheduler Proxy | The `ovirt-scheduler-proxy` package is removed in {virt-product-fullname} 4.4.z SP1.
+| oVirt Scheduler Proxy | The `ovirt-scheduler-proxy` package is removed in {virt-product-fullname} 4.4 SP1.
```

Comment:
Is there a way to get `RHV 4.4 SP1` vs `oVirt 4.5.0` here? otherwise here will display oVirt 4.4 SP1 which doesn't reflect oVirt versioning.

Reply:
@sandrobonazzola I've set it up so now it will show as 4.5 upstream and 4.4 SP1 downstream.

Reply:
I still see the issue here

Reply:
@sandrobonazzola Hi Sandro, According to peer review comments from Eli Marcus, the conditionalization is not needed in this instance. This is the release notes published in the downstream only; so it will NOT be included as part of  the upstream docs.

Reply:
ok, good to merge then.